src/neo/SmartContract/Native/ManagementContract.cs

```diff
@@ -12,7 +12,7 @@ namespace Neo.SmartContract.Native
 {
     public sealed class ManagementContract : NativeContract
     {
-        public override string Name => "Neo Contract Management";
+        public override string Name => "ContractManagement";
```

Comment:
rename the class too?

Reply:
Personally, I don't care. But yeah, I guess it should match. Given the naming of other native contracts, "ContractManagement" seems like a better choice than "ManagementContract", but as long as the name is a legal identifier I 100% do not care.